app/controllers/users_controller.rb

```diff
@@ -15,9 +15,17 @@ def admin_index
     authorize User
     if current_user.can_super_admin?
       @users = User.page(1)
+      @users_to_csv = User.all
     else
       @users = current_user.org.users.page(1)
+      @users_to_csv = current_user.org.users.order(:surname)
     end
+
+    respond_to do |format|
+      format.html
+      format.csv { send_data User::AtCsv.new(@users_to_csv).to_csv, filename: "users-accounts-#{Date.today}.csv" }
```

Comment:
Then once you've added the wrapper method that @Bodacious recommended, you can update this to something like:
```ruby
  format.csv { send_data @users.map(&:to_csv), filename: "users-accounts-#{Date.today}.csv" }
```

Reply:
I replaced this with `format.csv { send_data User.to_csv(@users), filename: "users-accounts-#{Date.today}.csv" }`